[DISCORD:1430520104801800263] Latest masta/core npm release broken

This issue was created from Discord post: https://discord.com/channels/1309558646228779139/1430520104801800263

[Error: Cannot find module '/home/ferreo/raven/raven/node_modules/@mastra/core/dist/ai-tracing/exporters/index.js' imported from /home/ferreo/raven/raven/node_modules/@mastra/langfuse/dist/index.js] {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///home/ferreo/raven/raven/node_modules/@mastra/core/dist/ai-tracing/exporters/index.js'
}
[Error: Cannot find module '/home/ferreo/raven/raven/node_modules/@mastra/core/dist/ai-tracing/exporters/index.js' imported from /home/ferreo/raven/raven/node_modules/@mastra/langfuse/dist/index.js] {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///home/ferreo/raven/raven/node_modules/@mastra/core/dist/ai-tracing/exporters/index.js'
}

The tracing exporters folder is missing the js files in @mastra/core 0.22.0 it seems

--- a/.changeset/small-rules-hide.md
+++ b/.changeset/small-rules-hide.md
@@ -1,0 +1,7 @@
+---
+'@mastra/braintrust': patch
+'@mastra/langsmith': patch
+'@mastra/langfuse': patch
+---
+
+Fix a `ERR_MODULE_NOT_FOUND` error that was caused by a bad import to `@mastra/core/dist/ai-tracing/exporters/index.js`

--- a/observability/braintrust/src/ai-tracing.ts
+++ b/observability/braintrust/src/ai-tracing.ts
@@ -6,10 +6,13 @@
  * Events are handled as zero-duration spans with matching start/end times.
  */
 
-import type { AITracingEvent, AnyExportedAISpan, LLMGenerationAttributes } from '@mastra/core/ai-tracing';
-import { AISpanType, omitKeys } from '@mastra/core/ai-tracing';
-import { BaseExporter } from '@mastra/core/ai-tracing/exporters';
-import type { BaseExporterConfig } from '@mastra/core/ai-tracing/exporters';
+import { AISpanType, omitKeys, BaseExporter } from '@mastra/core/ai-tracing';
+import type {
+  AITracingEvent,
+  AnyExportedAISpan,
+  LLMGenerationAttributes,
+  BaseExporterConfig,
+} from '@mastra/core/ai-tracing';
 import { initLogger } from 'braintrust';
 import type { Span, Logger } from 'braintrust';
 import { normalizeUsageMetrics } from './metrics';

--- a/observability/langfuse/src/ai-tracing.ts
+++ b/observability/langfuse/src/ai-tracing.ts
@@ -12,10 +12,13 @@
  * - Adapts to v5 streaming protocol changes
  */
 
-import type { AITracingEvent, AnyExportedAISpan, LLMGenerationAttributes } from '@mastra/core/ai-tracing';
-import { AISpanType, omitKeys } from '@mastra/core/ai-tracing';
-import { BaseExporter } from '@mastra/core/ai-tracing/exporters';
-import type { BaseExporterConfig } from '@mastra/core/ai-tracing/exporters';
+import { AISpanType, omitKeys, BaseExporter } from '@mastra/core/ai-tracing';
+import type {
+  AITracingEvent,
+  AnyExportedAISpan,
+  LLMGenerationAttributes,
+  BaseExporterConfig,
+} from '@mastra/core/ai-tracing';
 import { Langfuse } from 'langfuse';
 import type { LangfuseTraceClient, LangfuseSpanClient, LangfuseGenerationClient, LangfuseEventClient } from 'langfuse';
 

--- a/observability/langsmith/src/ai-tracing.ts
+++ b/observability/langsmith/src/ai-tracing.ts
@@ -6,10 +6,13 @@
  * Events are handled as zero-duration RunTrees with matching start/end times.
  */
 
-import type { AITracingEvent, AnyExportedAISpan, LLMGenerationAttributes } from '@mastra/core/ai-tracing';
-import { AISpanType, omitKeys } from '@mastra/core/ai-tracing';
-import { BaseExporter } from '@mastra/core/ai-tracing/exporters';
-import type { BaseExporterConfig } from '@mastra/core/ai-tracing/exporters';
+import { AISpanType, omitKeys, BaseExporter } from '@mastra/core/ai-tracing';
+import type {
+  AITracingEvent,
+  AnyExportedAISpan,
+  LLMGenerationAttributes,
+  BaseExporterConfig,
+} from '@mastra/core/ai-tracing';
 import type { ClientConfig, RunTreeConfig } from 'langsmith';
 import { Client, RunTree } from 'langsmith';
 import type { KVMap } from 'langsmith/schemas';

